src/2.6.0/wso2apim.js

```diff
@@ -247,6 +250,32 @@ function constructAPIDef(user, gatewayEnv, apiDef, apiId) {
   }
 }
 
+function constructCorsConfiguration(apiDef) {
+  const { origins, credentials, headers, methods } = apiDef.cors;
+  const defaultAllowHeaders /* default WSO2 cors config */ = [
+    'authorization',
+    'Access-Control-Allow-Origin',
+    'Content-Type',
+    'SOAPAction',
+    'apikey',
```

Comment:
Not sure if we should add `apikey` to the sane defaults. Unlike other 4 headers, it is not any standard header. Thoughts?

n.b. keep `authorization` -> `Authorization`

Reply:
I went with what WSO2 3.2.0 uses as default values if you enable CORS in the web interface. See screenshot. But I agree with you! I will change it. 
<img width="698" alt="image" src="https://user-images.githubusercontent.com/2051896/127830946-f3afdd62-62b1-4eb6-914a-406e4c43d903.png">